docs/data_access.rst

```diff
@@ -157,18 +157,33 @@ can also be installed using the conda package manager).
 If you're not familiar with using Unix command line tools in Windows you can also use a
 3rd party tool like `7zip <https://www.7-zip.org/download.html>`__.
 
-.. _access-zenodo:
+.. _access-stable:
 
 ---------------------------------------------------------------------------------------
-Zenodo
+Stable Builds
```

Comment:
Previously this section was trying to do double duty documenting both that the raw inputs are being archived on Zenodo, as well as the stable data releases.  The raw inputs part feels a little buried now. Should it go somewhere else entirely, since this page is more focused on accessing outputs?